state_version_integration_test.go

```diff
@@ -186,6 +186,8 @@ func TestStateVersionsCreate(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		// TODO: check state outputs for the ones we sent in JSONStateOutputs
```

Comment:
Can this TODO be resolved?

Reply:
No, not until the API changes are merged.